packages/kit/src/core/tsconfig.js

```diff
@@ -22,6 +22,19 @@ export function generate_tsconfig(config) {
 	/** @param {string} file */
 	const config_relative = (file) => path.relative(SVELTE_KIT, file);
 
+	const dirs = new Set([
+		project_relative(path.dirname(config.kit.files.routes)),
+		project_relative(path.dirname(config.kit.files.lib))
```

Comment:
Not sure if this works for everyone. People could have other stuff besides lib/routes below their src and want it to have included in their tsconfig. This also might exclude `app.d.ts` as a result. What's the reason for changing it from the previous implementation?

Reply:
Before, it was `../**/*.js` etc, which included stuff in things like `build` — it was causing our own CI to fail. This scopes it to `../src` by default (since that's the common parent of `src/routes` and `src/lib`), which works unless you start putting things outside `src` in which case you would need to modify your own `include` config (but hopefully at that point you know what you're doing)

Reply:
Ah I missed the `path.dirname`, I thought this would result in `src/lib/**/*.ts` etc, but it results in `src/**/*.ts` - all good then 👍